### PR TITLE
Only use Mono_Printf when necessary

### PR DESCRIPTION
**Pull request by abergmeier**
`Mono_Printf("%s", s)` seems to make no sense and thus turn into Mono_Print.
Motivation behind this change is that if I want to change behavior or e.g. `Mono_Printf` I only want to look at the code that actually needs printf and not all the rest.

### CODE REVIEW

File: .github/workflows/win.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      uses: microsoft/setup-msbuild@v1.0.0
+    - name: Install Windows 8.1 SDK
+      shell: powershell
+      run: |
+        Invoke-WebRequest -Method Get -Uri https://go.microsoft.com/fwlink/p/?LinkId=323507 -OutFile sdksetup.exe -UseBasicParsing
+        Start-Process -Wait sdksetup.exe -ArgumentList "/q", "/norestart", "/features", "OptionId.WindowsDesktopSoftwareDevelopmentKit", "OptionId.NetFxSoftwareDevelopmentKit"
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Compile
+      run: msbuild CnCRemastered.sln
```
Comment (elcuco) [inline .github/workflows/win.yaml:19]: This file should be in another PR

Comment (abergmeier) [reply .github/workflows/win.yaml:19]: It is. I just wanted CI to check whether it still compiles.

**Comment by abergmeier** [comment]
> Nothing wrong with the patch itself... but I don't undestand it. Does this project aim to improve the original code, or just make it compile? If we are only to make this compile, then - this PR should be ignored.

It is somewhere in between. It makes porting the code easier.

File: .github/workflows/win.yaml (hunk 1 above)
Comment (elcuco) [reply .github/workflows/win.yaml:19]: you are not compiling locally?

Comment (abergmeier) [reply .github/workflows/win.yaml:19]: Only have Unix machines here :) So no, I need a CI to check Windows for me.
